docs/en/relying-party-solution.rst

```diff
@@ -0,0 +1,308 @@
+    - **Verifier (FE)**
+    - **Verifier (BE)**     
+    - Request a Session Cookie through polling, which will be available downstream after the checks performed in the previous step
+  * - **17**
+    - **Verifier (BE)**
+    - **-**     
+    - Consume the VP Token
+  * - **18**
+    - **Verifier (BE)**
+    - **-**     
+    - Requests the invalidation of the current session as the transaction has been successfully completed
+  * - **19**
+    - **Verifier (FE)**
+    - **User**     
+    - The user is successfully authenticated on the Verifier's website
+
+
+AuthorizationRequest Details
+----------------------------
+The AuthorizationRequest object is required by the Relying Party FrontEnd in order to show the QR Code. 
+The User will scan the QR Code using the Wallet granting the attributes to the RP.
+
+.. code-block:: javascript
+
+  POST /oid4vp HTTP/1.1
+  HOST: <VERIFIER_RELYING_PARTY_HOST>
+
+  HTTP/1.1 201 OK
+
+
+Below a non-normative response example
+
+.. code-block:: javascript
+
+  {
+    "alg": "ES256",
+    "typ": "JWT",
+    "x5c": [ "MIICajCCAdOgAwIBAgIC...20a" ],
+    "kid": "e0bbf2f1-8c3a-4eab-a8ac-2e8f34db8a47",
+    "trust_chain":[
+      "MIICajCCAdOgAwIBAgIC...awz",
+      "MIICajCCAdOgAwIBAgIC...2w3",
+      "MIICajCCAdOgAwIBAgIC...sf2"
+    ]
+  }
+  .
+  { 
+    "transactionId": "e9673f90-a652-4c47-8039-f60096cabafb",
+    "state": "a1252f90-a652-4c47-8039-f60096cavzqw",
+    "nonce": "o1923f90-a652-4c47-8039-f60096cabcza",
+    "requestUri": "oid4vp-wallet://authorize?client_id=https://<VERIFIER_RELYING_PARTY_HOST>&request_uri=https://<VERIFIER_RELYING_PARTY_HOST>/oid4vp/a1252f90-a652-4c47-8039-f60096cavzqw",
+    "iss": "https://<VERIFIER_RELYING_PARTY_HOST>",
+    "aud": "https://<VERIFIER_RELYING_PARTY_HOST>",
+    "iat": "1686823112",
+    "exp": "1686823812"
+  }
+  .
+  SIGNATURE
+
+
+Header
+^^^^^^
+
+.. list-table:: 
+  :widths: 25 50
+  :header-rows: 1
+
+  * - **Param**
+    - **Description**
+  * - **alg**
+    - Algorithm used to sign the JWT
+  * - **typ**
+    - Media Type of the JWT
+  * - **x5c**
+    - X.509 Certitifcate Chain containing the Key used to sign the JWT
+  * - **kid**
+    - Key ID used identifying the Key used to sign the JWT. Required if trust_chain is used
+  * - **trust_chain**
+    - Sequence of verified Entity Statements that validates a participant's compliance with the Federation
+
+
+Payload
+^^^^^^^
+.. list-table:: 
+  :widths: 25 50
+  :header-rows: 1
+
+  * - **Param**
+    - **Description**
+  * - **transactionId**
+    - Unique identifier of the authorization transaction
+  * - **state**
+    - Unique identifier of the AuthorizationRequest
+  * - **nonce**
+    - Fresh cryptographically random number with sufficient entropy used for Security Reason
+  * - **requestUri**
+    - The payload of the QR Code from where the Wallet can obtain the Request Object containing Authorization Request parameters
+  * - **iat**
+    - The timestamp when the JWT was issued
```

Comment:
timestamp, we should say unix timesamp or numericDate according to ISO ...

Reply:
Done